src/content/en/updates/2011/11/Quota-Management-API-Fast-Facts.md

```diff
@@ -8,46 +8,45 @@ book_path: /web/updates/_book.yaml
 # Quota Management API : Fast Facts {: .page-title }
 
 {% include "web/_shared/contributors/agektmr.html" %}
+November 2011
```

Comment:
We do not put date on top of the article at the moment. 
Is there reason for adding date here ??